src/aap_eda/core/validators.py

```diff
@@ -45,7 +45,7 @@ def check_if_de_exists(decision_environment_id: int) -> int:
     try:
         de = models.DecisionEnvironment.objects.get(pk=decision_environment_id)
         if de.eda_credential_id:
-            models.EdaCredential.objects.get(pk=de.eda_credential_id)
+            check_if_de_valid(de.image_url, de.eda_credential_id)
     except models.EdaCredential.DoesNotExist:
```

Comment:
seems we longer need to catch this exception since `get_credential_if_exists()` does it already

Reply:
removed